incubator/hnc/pkg/reconcilers/hnc_config_test.go

```diff
@@ -28,6 +39,9 @@ var _ = Describe("HNCConfiguration", func() {
 	)
 
 	BeforeEach(func() {
+		// Increase the timeout of `Eventually` for test cases in this file because the `Reconcile`
+		// method of ConfigReconciler sleeps at the very beginning of the method.
+		SetDefaultEventuallyTimeout(4 * time.Second)
```

Comment:
You're already overriding Eventually with the constants you've defined above, why do you need this too?

Reply:
Ditto. Resolve this comment since it is outdated.